src/App.tsx

```diff
@@ -1,85 +1,93 @@
 import React from 'react';
+import classNames from 'classnames';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import goods from './api/goods';
 
-export const goods = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
-];
+type State = {
+  status: string;
+};
```

Comment:
from `status` I expect to see something like `completed, rejected` or `healthy, in_error` or even http responds status code

but I saw 'Jam' there... please use more clear variable names 🙂